src/Mod/Sketcher/Gui/TaskSketcherElements.cpp

```diff
@@ -486,7 +488,39 @@ ElementItem* ElementItemDelegate::getElementtItem(const QModelIndex& index) cons
     ElementView* elementView = static_cast<ElementView*>(parent());
     return elementView->itemFromIndex(index);
 }
-// ----------------------------------------------------------------------------
+
+/* Filter element list widget ------------------------------------------------------ */
+ElementFilterList::ElementFilterList(QWidget* parent) : QListWidget(parent)
+{
+    for (auto const &filterItem:filterItems) {
+        auto it = new QListWidgetItem(tr(filterItem));
+        it->setFlags(it->flags() | Qt::ItemIsUserCheckable);
+        it->setCheckState(Qt::Checked);
+        addItem(it);
+    }
+}
+
+ElementFilterList::~ElementFilterList()
+{
+}
+
+void ElementFilterList::changeEvent(QEvent* e)
+{
+    if (e->type() == QEvent::LanguageChange) {
+        languageChange();
+    }
+    QWidget::changeEvent(e);
+}
+
+void ElementFilterList::languageChange()
+{
+    assert((int)filterItems.size() == count());

```

Comment:
If you're going to leave this in, I'd prefer to see a `static_cast`, in part just to keep the linter quiet.